lib/tests-from-pull-request.js

```diff
@@ -28,29 +34,32 @@ function getTestsFromPR(context) {
   }
 }
 
-function getTestNames(prPatch) {
+function getTestNames(prPatch, config) {
   const testNames = [];
   const parsedPatch = prPatch.split('\n');
-
   parsedPatch.forEach((line) => {
     if (line.startsWith('+')) {
       line = removePrefixAndTrim(line);
-      const testName = getTestName(line);
+      const testName = getTestName(line, config);
 
       testName ? testNames.push(testName) : null;
     }
   });
-
   return testNames;
 }
 
 function removePrefixAndTrim(line) {
   return line.slice(1).trim();
 }
 
-function getTestName(line) {
-  const getTestNameRegex = /it\((.*),/;
+function getConfig(context) {
+  return Promise.resolve(context.config('config.yml'));
+}
+
+function getTestName(line, config) {
+  const testFunctionName = _.get(config, 'tdd1tTestFunctionName', 'it');
+  const getTestNameRegex = new RegExp(`${testFunctionName}\\((.*),`);
   const regexTestResult = getTestNameRegex.exec(line);
 
-  return regexTestResult && regexTestResult[1].trim().slice(1,-1);
+  return regexTestResult && regexTestResult[1].trim().slice(1);
```

Comment:
removing the `-1` introduced a bug here.

if you run the code on the following line:
```
it("should do it", () => {})
```
the `regexTestResult[1]` will be equal `"should do it"`.

removing just the first character will result in `should do it"` while removing both the first and the last with `.slice(1, -1)` makes sure we remove both quotes.

Reply:
Well, apparently I forgot the closing `"` when testing :see_no_evil: